package.json

```
@@ -6,7 +6,8 @@
   "changelogUrl": "https://github.com/DeNA/Anjin/releases",
   "dependencies": {
     "com.cysharp.unitask": "2.3.3",
-    "com.nowsprinting.test-helper.monkey": "0.1.2",
+    "com.nowsprinting.test-helper.monkey": "0.5.0",
+    "com.nowsprinting.test-helper.random": "0.1.0",
```

```suggestion
    "com.nowsprinting.test-helper.random": "0.2.0",
```
So strictly. There is no problem because the test-helper.monkey dependency solves it.